gateway/controller_test.go

```diff
@@ -155,9 +158,9 @@ var _ = ginkgo.Describe("Controller", func() {
 		Ω(err).Should(BeNil())
 
 		c.SetNamespace("ns1")
-		tablesGot, err = c.FetchAllSchemas()
+		tableAddressesGot, err := c.FetchAllSchemas()
 		Ω(err).Should(BeNil())
-		Ω(tablesGot).Should(Equal(tables))
+		Ω(tableAddressesGot).Should(Equal(tableAddresses))
```

Comment:
better to add another test case with two table

Reply:
done